src/99-appendices/08-coordinate-systems.md

```diff
@@ -1,5 +1,49 @@
 # Appendix VIII: Preferred Names of Coordinate Systems
 
+## Definitions
+
+### Coordinate System
+
+A coordinate system specifies the information necessary to interpret the
+coordinates associated with the data. A coordinate system can be described by
+its topology (cartesian, spherical, polar, etc.,.), dimensionality, and units of
+measurement along each dimension. Every device that makes spatial measurements
+or has sensors described in a space has a coordinate system (e.g., an MR
+scanner, an MEG scanner, an EEG cap, Polhemus digitizer). The data output by
+most of these devices describe the number of dimensions, the units along each
+dimension, and in some cases the details of origin (for example, time).
+
+### Space, atlas and map
+
+A space/atlas/map is embedded in a coordinate system to ascribe meaning to the
+coordinates. For instance, Scanner, Talairach88, MNI, FS are different
```

Comment:
Here it is mentioned that 'Scanner' is a coordinate system, but it is not listed in the label below Volume. Should we use 'Other' to describe scanner coordinates?

Reply:
I agree that "scanner" is not a good name. Elsewhere we use "other" and then require the detailled specification in the _coordsystem.json file. And [elsewhere](
https://github.com/bids-standard/bids-specification/pull/109#issuecomment-471099335)  "orig" is used.

I would consider "device" a good name. It generalizes to different devices and toes not make implicit assumptions about temporal order (as is done in "orig", which is prone to be interpreted differently by different people).

Reply:
From my current reading of the contents of this PR the section "Definitions" in Appendix VIII that this discussion revolves around is not crucial for clarifying the use of any of the data types described elsewhere in the PR. Perhaps we could move it to a new PR and work on it on an independent schedule?

Reply:
As a comment from the meta-level: I must admit that I find it very hard to keep the of the PRs goals and content on github (in general, also this one).  I also fear that others with even more limited github skills than me might not be able to contribute. 

The changes in this PR relate to specification texts  that are not restricted to (f)MRI. That is why I am commenting to `derivatives/01-introduction.md` (which will in the future also apply to meg/ieeg/eeg), common-data-types (it does not define scope and does not say "MR specific data types", so I guess I'll have to deal with this in the future as well), and a "space/coordinates" appendix that underlies not only derivatives, but also the raw data specification and that actually originated from MEG and now gets used for MRI derivatives as well. 

I would not object if those three (or two, in case you consider document `02` to be `mr-specific-data-types`) were discussed in their own PRs.

Reply:
Yeah, it's challenging. We have been using smaller PRs to this PR to structure work, but it's not perfect. I don't think that splitting introduction, the few necessary changes to Appendix VIII, and actual new data types (MR derivatives) into separate independently mergeable PRs would be good. The new data types drive the "common" rules, and the "common" rules need to work with the new data types. Those need to be considered together. Of course, it also sets the ground for future derivatives - that's why your input is so important and why I appreciate you monitor this PR. The sections you focus on are the best use of your time.

What I was proposing in the previous comment was to shrink the scope of this PR by moving the "Definitions" section of https://github.com/bids-standard/bids-specification/blob/c42c4bbb9a84886740e8c2583575ae68363c2d15/src/99-appendices/08-coordinate-systems.md to a separate PR. Its content does not influence much of the rest of the PR and it seems to be causing some controversy that can be resolved on a different schedule.

Reply:
Being clear about how to refer to scanner space does seem relevant for the current data type (MRI/DTI). For example: fiber pathways have x,y,z coordinates, per these derivatives, do they always have to be defined in a normalized/ACPC space or could they potentially live in scanner space? 
(The fact that we also define iEEG electrode coordinates as the center of mass of objects from a T1w image is a side-note, but we do want to write code to integrate DTI/MRI/iEEG data)

Reply:
at least in MR the coordinate system is encoded (somewhat) in the nifti header of the raw data. `space-orig` was intended to capture that a derivative contains information in the same "space" as the original file from which it was derived. if there is an intermediate alignment step, then the ReferenceMap would point to a realigned file.

Reply:
Indeed - I wasn't clear. Specifying what coordinate system derivatives are is important and we are working hard on refining this language (see for example https://github.com/chrisfilo/bids-specification/pull/12/files), However the extra "definitions" section in Appendix VIII is not strictly necessary for this purpose (as it wasn't for `space` in context of iEEG). I think it's a useful addtion, but could be discussed in a separate PR.

Reply:
Thanks for clarifying - is it then the case that ReferenceMap is similar to how we use IntendedFor in _coordsystem.json, and the <label> to use in space-<label> for scanner space is 'Other'?
(I would not like to start naming my ieeg files _space-Other and then other derived data _space-SomethingElseToDescribeScannerSpace)

Reply:
to keep the threads on topic I replied at https://github.com/chrisfilo/bids-specification/pull/12#issuecomment-473985746